packages/gap-web-ui/src/components/table/Table.module.scss

```diff
@@ -16,3 +16,7 @@
 .gap-table-cell-wrap-text {
   word-break: break-word;
 }
+
+.disable-bottom-row-border {
+  border-bottom: none !important;
```

Comment:
is there any way we can do this without !important?

Reply:
I'll play about with it again but I don't believe so